bin/smack.py

```diff
@@ -254,7 +260,16 @@ def clang_frontend(args):
   smack_bc = temporary_file('smack', '.bc', args)
   try_command(compile_command + [smack_lib(), '-o', smack_bc])
   try_command(compile_command + [args.input_file, '-o', args.bc_file])
-  try_command(['llvm-link', args.bc_file, smack_bc, '-o', args.bc_file])
+  link_targets = [args.bc_file, smack_bc]
```

Comment:
Mike (and Z...), as we add more features to SMACK, we'll probably end up needing to support additional SMACK provided libraries.  We might want to come up with a less hard coded way of compiling and linking SMACK provided libraries like these.